pkg/omf/omf.fish

```diff
@@ -119,13 +123,18 @@ function omf -d "Oh My Fish"
 
     case "t" "theme"
       if test (count $argv) -eq 1
-        set -l ostype (uname)
         set -l theme (cat $OMF_CONFIG/theme)
-        set -l regex "[[:<:]]($theme)[[:>:]]"
-        test "$ostype" != "Darwin"; and set regex "\b($theme)\b"
-
-        omf.packages.list --database --theme | column | sed -E "s/$regex/"(omf::em)"\1"(omf::off)"/"
-        omf::off
+        set -l highlight_current "(^|\s)($theme)(\s|\$)"
+        # regex for wrapping everything else with (omf::off), just in order to keep the alignment
+        set -l normal_item "\S+"
+
+        echo (omf::under)"Installed:"(omf::off)
+        # align before adding color codes. otherwise column messes the alignment
+        # this marks the current theme twice, both with 'off' and 'dm', but this "mistake" actually keeps the alignment better for some reason
+        omf.packages.list --installed --theme | column | sed -E s/"$highlight_current"/"\1"(omf::em)"\2"(omf::off)"\3"/g | sed -E s/"$normal_item"/(omf::off)"\0"(omf::off)/g
```

Comment:
See @derekstavis's screenshot above (one of the first in this PR). Alignment with highlighting was a tough nut and so far that's the only way I managed to make it work. Spent way too many hours and eventually the solution is a ugly hack. Maybe one day I will write a `column` alternative in Fish...